src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import '../../../../shared/css/nested-menu.scss'
+import '../../../AccountHeader/styles.scss'
+import '../../../AccountHeader/balance-selector.scss'
+import { localizeBalance, localizeNumber } from '../../../../shared/utils'
+import Currency from '../../../../shared/components/Currency'
+
+export interface AmmDataType {
+  balance: { currency: string; amount: number; issuer: string }
```

Comment:
Isn't this already a type somewhere? called Amount or something?

Reply:
no, we already had this discussion.